src/ui/buttons/button.jsx

```diff
@@ -111,9 +111,19 @@ export function Button({
 
   // if no color option is passed in via style props
   if (color === "" || typeof color === "undefined") {
-    // if multiple buttons are being rendered (smart stack), we set default color as gold
     // if a single button is rendered, we set color to first option in the fundingSource config
-    color = multiple ? "gold" : colors[0];
+    color = colors[0];
+
+    // if multiple buttons are being rendered (smart stack), we set default color as gold > first
```

Comment:
Does this assumption break with any of the other buttons? I feel like it's something we could accidentally break with a different button in the future.

Is the order of colors in the config too intertwined with other logic to prevent switching around the order to put the default first? Possibly risky given the lack of test coverage around this.

Reply:
When `multiple === true` , we will use the `secondaryColor` in the funding config to decided the color of the button in the smart stack.

i.e. when you pass in `color: gold`, the PayPal button & PayLater button will actually be gold, but the Venmo button gets set to `blue`, Credit gets set to `darkblue` and Card gets set to `black`

I think a lot of the confusion lies around the `secondaryColors`. We can't simply set the default color to `gold` since not all buttons have `gold` as a valid color, but we need `gold `to be set as the default color to correctly look up the color to render in a button in the smart stack using the `secondaryColors['gold']`

The bug was caused by `color === 'gold'` on a button that does not have `gold` as an available color. If we can be sure we're not going to run into this scenario, we should be good to go. Then we can come back and refactor this logic once we have solid testing in place for validating the correct class is applied to the buttons.